Sources/Core/AWSClientRuntime/Config/AwsDefaultClientConfiguration.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+public protocol AwsDefaultClientConfiguration {
```

Comment:
nit: Swift convention is to leave acronyms all-caps in CamelCase.

So, `AWSDefaultClientConfiguration` not `AwsDefaultClientConfiguration`